app/renderer/js/components/tab.ts

```diff
@@ -1,9 +1,12 @@
 'use strict';
 
-const BaseComponent = require(__dirname + '/../components/base.js');
+import BaseComponent = require('../components/base');
 
 class Tab extends BaseComponent {
-	constructor(props) {
+	props: any;
+	webview: any;
```

Comment:
We should also figure out type of this. Shouldn't be `any`.

Reply:
Ah, so this should be `WebView` class.

Reply:
Add a todo comment here to remind use that we need to annotate this to `WebView`.

Reply:
@priyank-p I have tried using WebView here. Please check and tell if it is the correct way or not.